src/components/cylc/table/Table.vue

```diff
@@ -99,6 +99,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
           fluid
           class="ma-0 pa-0 w-100 h-100 left-0 top-0 position-absolute"
         >
+          {{  }}
```

Comment:
@AaronDCole Why add this?

Reply:
Ahhh no idea? I think thats a mistake, ill remove it